src/test/java/org/inferred/freebuilder/processor/RequiredPropertiesTest.java

```diff
@@ -248,6 +302,129 @@ public void testBuilderGetterThrowsIfValueUnset() {
         .runTest();
   }
 
+  @Test
+  public void testMergeFromBuilder_defaultsDoNotOverride() {
```

Comment:
This looks identical to a test in `DefaultedPropertiesTest`, up to using `DEFAULTED_PROPERTIES_TYPE`. Should this test `REQUIRED_PROPERTIES_TYPE`?